<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDEPARTMENT</class>
 <widget class="QWidget" name="CDEPARTMENT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>320</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>部门编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>部门领导</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>380</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>上级部门</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="No_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leader_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="parent_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnSave">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnClose">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>450</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="departmentName">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>45</y>
     <width>690</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>290</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>部门名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
